DataFormats/GEMDigi/BuildFile.xml

```diff
@@ -1,4 +1,5 @@
 <use   name="DataFormats/Common"/>
+<use   name="DataFormats/GeometryVector"/>
```

Comment:
my previous comment to this file related to all changes.
There are no other changes to this package ( DataFormats/GEMDigi). Why does the build file need a change?
If not needed, please remove.